module1/src/css/App.scss

```diff
@@ -0,0 +1,47 @@
+.header {
```

Comment:
don't mix scss and css, rename this folder as: styles
also this file contains all styles for all components, create separated folders for each component with styles, index.js etc..